package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "webdb-challenge",
+  "version": "1.0.0",
+  "description": "This challenge allows you to practice the concepts and techniques learned over the past Sprint and apply them in a concrete project.",
+  "main": "index.js",
+  "scripts": {
+    "server": "nodemon index.js",
+    "start": "java index.js"
```

Comment:
In your start script you have java, however we are working with node, you should fix that script.